README.md

```diff
@@ -57,6 +57,23 @@ Then you can run the utility:
 plugincheck2 -sourceCodeUri [source_code_location/] [plugin_archive.zip]
 ```
 
+### NPX With Local Files
```

Comment:
How about renaming the section: "Generating local files for validation"?

Other than the npx cmd itself, the steps - I believe - would be applicable for users of docker or locally built versions of the tool?

Also just to check - is there anything in the other plugin tools (create/sign) that would help generate the zip file?

cc @academo

Reply:
> How about renaming the section: "Generating local files for validation"?

Done.

> Also just to check - is there anything in the other plugin tools (create/sign) that would help generate the zip file?

@academo should answer this one, but as far as I know, no. The `release.yml` workflow for the starter plugin uses the `mv` and `zip` commands to package the plugin. I would have assumed if there was another plugin tool to do this, it would have been used there.

Reply:
There's no single command to build and create the zip file.